mautic/mautic
# Allow using classes in {mauticform element=button-selector}

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? |  Yes
| New feature? | 
| MauticBox url | https://mautibox.com/7510/page/preview/4#
| Related user documentation PR URL | https://github.com/mautic/documentation/pull/305
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | 
| BC breaks? | 
| Deprecations? |  

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:
This PR allowed using selectors which point to multiple elements in `{mauticform}` token.
##### Before (supported only single element link):
```html
<a class="btn btn-default" href="#" id="call-to-action-one">Click me</a> 
{mauticform id=2 style=modal element=#call-to-action-one} 

<a class="btn btn-default" href="#" id="call-to-action-two">Click me</a> 
{mauticform id=2 style=modal element=#call-to-action-two} 
```

##### After (classes along with multiple elements can be used):
```
<a class="btn btn-default call-to-action-btn" href="#">Click me</a> 
<a class="btn btn-default call-to-action-btn" href="#">Click me</a> 

{mauticform id=2 style=modal element=.call-to-action-btn} 
```


#### Steps to test this PR:
1. Load up [this PR](https://mautibox.com)
2. Put a form snippet on a Landing Page, e.g {form=2}
3. Make sure that Before(selecting one button by id) and After (selecting multiple buttons by classes) cases work.
